spring-integration-core/src/main/java/org/springframework/integration/support/management/graph/IntegrationNode.java

```diff
@@ -47,8 +47,11 @@ protected IntegrationNode(int nodeId, String name, Object nodeObject, Stats stat
 		this.componentType = nodeObject instanceof NamedComponent ? ((NamedComponent) nodeObject).getComponentType()
 				: nodeObject.getClass().getSimpleName();
 		this.stats = stats;
-		if (nodeObject instanceof ExpressionBased) {
-			this.properties.put("expression", ((ExpressionBased) nodeObject).getExpressionString());
+		if (nodeObject instanceof IntegrationObjectSupport) {
```

Comment:
I'll restore the interface, but `ExpressionBased` is probably not the right name, now though.